src/features/input/InputView.vue

```diff
@@ -396,8 +399,8 @@
             <v-icon class="pb-3" small v-on="on">mdi-help-circle</v-icon>
           </template>
           <span
-            >A sampling strategy to find suggested points for additional
-            measuring.</span
+            >A sampling strategy to find suggested <br />
+            points for additional measuring.</span
```

Comment:
The sampling strategy to be employed to find locations where additional training datsets might be usefull.